scripts/Jenkinsfile

```diff
@@ -170,6 +185,26 @@ pipeline {
                         }
                     }
                 }
+                stage("Upload PyPi Legion packages") {
+                    steps {
+                        script {
+                            if (env.param_stable_release && env.param_upload_legion_package_pypi) {
+                                def im = docker.build("python-alpine", "${pythonAlpinePath}")
+                                im.inside{
+                                    legion.publishPythonPackage("./", env.param_pypi_repository)
+                                }
+                            }
+                            else {
+                                if (!env.param_stable_release){
+                                    print("Publishing packages to public registry skipped – not stable release")
+                                }
+                                else{
+                                    print("Publishing packages to public registry skipped – publishing option is disabled")
```

Comment:
We should push in the nexus for every build and pypi\npm\... for every *stable* release.